### ISSUE TITLE
Request

### ISSUE THREAD

**Issue by DatTango** [FBanna/ChestProtection#3]
Sorry, I didn't know where to put requests, I added this to my server and all my friends **LOVED** it, only request they had was if they could add a way to add like whitelists to chests, maybe like write their name in the book or some way to whitelist them to let them open the chest, Thank you for hearing me out 🙏

Also we haven't had this problem yet but I'd imagine eventually people will figure that they can trap player by locking chests because they can't break it, would be nice if server owner could config some stuff about it. 

These are just two inputs I had, I love your mod and just wanted to give ideas on how to make it even better, thank you. 🥰

**Comment by FBanna** [comment]
Yeah, I've thought of this stuff before and definitely need to add them. Do have any idea of how to stop traps etc. from being possible with them...? Also they could be used to trap withers and things which isn't vanilla at all. Any ideas would be great. I could make it so OP players could break chests still  but ive not thought of a good way of solving the problem

**Comment by DatTango** [comment]
Maybe make it to where if a player loads in near a chest or is near a chest for a extended period of time and tries to break it, the chest becomes transparent for a bit letting the dropped player leave or for the wither you can make an exception to become transparent if mobs try to break it

**Comment by FBanna** [comment]
A bit late but the new version now has authorisation to add players to the whitelist. Unfortunately I am still unsure of what I can do with trapping players. Because it is a server-side only mod I can not fade out blocks although I now have a config to allow moderators to change locks and shops

Hopefully the new 2.0.0 fixes your issues, tell me if you have any problems with it!
